### DISCUSSION TITLE
[Question] Long Term Support

### DISCUSSION THREAD

**Discussion by MikeRzDev** [mobxjs/mobx.dart#763] [General]
Hello, I have enjoyed working with mobx.dart in my first flutter project but I would like to know if this library would be supported for at least 3 more years, I don't want to sound rude, and I appreciate all the effort put into this project, but it seems that there are not new features ahead and I dont see any roadmap or milestones for the future, so I'm wondering if this project will be maintained or archived. thank you.

**Comment by pavanpodila** [comment]
Thanks Mike. The project will definitely be maintained. We have many users here actively using it. Agree on the roadmap.

This needs to become a community driven effort. I'm happy to collaborate. If you want to take a stab at listing out your own needs, we can build up from there.

**Comment by Rudiksz** [comment]
I have been using mobx (and following the progress on and off since 2 years) and I can't think of a feature that was added that I actually used, or one I would actually need. That 's not to say other people can't have different needs or there's nothing to improve. I'm just saying that lack of new features can be a sign of stability and fitness - as in: it does what it said it would do.

Here's a simple package I wrote for an infintely nestable and resizeable panel system using mobx.
https://pub.dev/packages/tabpanel

Just for the sake of comparison, I tried to implement this using the other big "state managaers" (which I won't name), and I gave up. It was seriously much more cumbersome to work with them.

My implementation is not fully "declarative", because not everything is a widget in it, so it's definitely not for the Flutter purists. I'm just plugging it as an example for what mobx can do.
